src/main/webapp/app/exam/manage/students/upload-images/students-upload-images-dialog.component.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, Input, OnDestroy, ViewChild, ViewEncapsulation } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { AlertService } from 'app/core/util/alert.service';
+import { HttpErrorResponse } from '@angular/common/http';
+import { Subject } from 'rxjs';
+import { ActionType } from 'app/shared/delete-dialog/delete-dialog.model';
+import { Exam } from 'app/entities/exam.model';
+import { ExamManagementService } from 'app/exam/manage/exam-management.service';
+import { faArrowRight, faBan, faCheck, faCircleNotch, faSpinner, faUpload } from '@fortawesome/free-solid-svg-icons';
+import { onError } from 'app/shared/util/global.utils';
+
+interface NotFoundExamUserType {
+    numberOfUsersNotFound: number;
+    numberOfImagesSaved: number;
+    listOfExamUserRegistrationNumbers: string[];
+}
+@Component({
+    selector: 'jhi-student-upload-images-dialog',
+    templateUrl: './students-upload-images-dialog.component.html',
+    styleUrls: ['./students-upload-images-dialog.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+})
+export class StudentsUploadImagesDialogComponent implements OnDestroy {
```

Comment:
Missing test class for this component.

Reply:
thanks for pointing this out, as we need to merge the PR asap, @pellumbbaboci please add the test in a follow-up PR